src/lib/dnssd/minimal_mdns/QueryReplyFilter.h

```diff
@@ -70,8 +70,22 @@ class QueryReplyFilter : public ReplyFilter
         {
             return true;
         }
-
-        return ((mQueryData.GetType() == QType::ANY) || (mQueryData.GetType() == qType));
+        if ((mQueryData.GetType() == QType::ANY) || (mQueryData.GetType() == qType))
```

Comment:
How about #12413 - that  one is doing the same thing  and allowing anything for additional (and the rules on what to send additional are encoded in the responder setup).

Is this change still needed?